sdk/trace/src/main/java/io/opentelemetry/sdk/trace/ReadableSpan.java

```diff
@@ -78,4 +80,8 @@
    * @return the kind of the span.
    */
   SpanKind getKind();
+
+  /** Returns the value for the given {@link AttributeKey}, or {@code null} if not found. */
```

Comment:
I think it's worth a note here about the fact that this call does require some locking, and that subsequent calls may result in different results, due to the inherent mutability of the underlying object.

Reply:
do you want me to add that same note for `hasEnded()`, `getName()`, and `getLatencyNanos()` also?

Reply:
that's actually probably a good idea, yes, if you don't mind.